### DISCUSSION TITLE
Welcome to ServoPixelDisplay Discussions!

### DISCUSSION THREAD

**Discussion by aaronse** [aaronse/servo-display#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Salvolongo** [comment]
hi, I tried your program but the servomotors move jerkily, what can I do to improve the movement of the motors? Furthermore, I would like to slow down the speed of the servomotors, what can I do?

**Comment by aaronse** [comment]
Hi @Salvolongo, followed up in https://github.com/aaronse/ServoPixelDisplay/discussions/9.  Cheers!
